android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/RepeatingGroupFactory.java

```diff
@@ -338,7 +335,8 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
             @Override
             public void afterTextChanged(Editable s) {
                 doneButton.setImageResource(R.drawable.ic_done_grey);
-                addOnDoneAction(referenceEditText, doneButton, widgetArgs);
+                if (widgetArgs.getJsonObject().optBoolean(JsonFormConstants.EXPAND_ON_TEXT_CHANGE, false))
```

Comment:
We can simplify to `optBoolean(JsonFormConstants.EXPAND_ON_TEXT_CHANGE)` instead of `optBoolean(JsonFormConstants.EXPAND_ON_TEXT_CHANGE, false))`.